src/features/hotspots/root/WelcomeOverview.tsx

```diff
@@ -78,6 +82,54 @@ const WelcomeOverview = ({ accountRewards }: Props) => {
     (state: RootState) => state.validators.validators.loading,
   )
 
+  const accountAddress = useSelector(
+    (state: RootState) => state.account.account?.address,
+  )
+
+  const hotspotRewards = useSelector(
+    (state: RootState) => state.hotspots.rewards || {},
+  )
+
+  const currentOraclePrice = useSelector(
+    (state: RootState) => state.heliumData.currentOraclePrice,
+  )
+
+  // Hook that is used for helium balance widget.
+  useAsync(async () => {
+    const token = await getWalletApiToken()
```

Comment:
Should these effects be running on Android too? I was thinking to add a check here `if (Platform.OS === 'android') return` but it looks like that crashes the app on iOS which is interesting.

Reply:
I was able to wrap the logic in the `useAsync` hooks with `if (Platform.OS === 'ios')` which should be fine